src/main/java/com/example/java/test/junior/developer/dto/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package com.example.java.test.junior.developer.dto;
+
+import lombok.*;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+@AllArgsConstructor
+@Builder
+public class UserRequestDto {
+
+    @NonNull
+    private final Long id;
+
+    @NonNull
+    private final String firstName;
+
+    @NonNull
+    private final String lastName;
+
+    @Email(message = "Email should be valid")   // Використовувати javax.validation для верифікації ?
```

Comment:
This is good annotation, but unfortunately doesn't work out of the box. You need to specify regex by your own. 
`@Email(regexp = "^[\\w.%+-]+@[\\w.-]+\\.[a-zA-Z]{2,}$")`